cloud/vsphere/deployer.go

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This file does none of the deploying. Maybe rename to something like DeploymentUtils?

Reply:
Updated the name to reflect is a client for vsphere deployement. Hopefully that is better?